packages/rspeedy/create-rspeedy/template-react-ts/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "jsx": "preserve",
+    "jsx": "react-jsx",
```

Comment:
I think using `"jsx": "react-jsx"` without `"jsxImportSource"` would also cause type error if using the same `@types/react` with Web projects.